force-app/main/default/lwc/wireGetPicklistValues/__tests__/wireGetPicklistValues.test.js

```diff
@@ -0,0 +1,57 @@
+import { createElement } from 'lwc';
+import WireGetPicklistValues from 'c/wireGetPicklistValues';
+import { registerLdsTestWireAdapter } from '@salesforce/lwc-jest';
+import { getPicklistValues } from 'lightning/uiObjectInfoApi';
+
+const mockGetPicklistValues = require('./data/getPicklistValues.json');
+
+// Register as an LDS wire adapter. Some tests verify the provisioned values trigger desired behavior.
+const getPicklistValuesAdapter = registerLdsTestWireAdapter(getPicklistValues);
+
+describe('c-wire-get-picklist-values', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    describe('getPicklistValues @wire data', () => {
+        it('renders seven lightning-input fields of type checkbox', () => {
+            // Create element
+            const element = createElement('c-wire-get-picklist-values', {
+                is: WireGetPicklistValues
+            });
+            document.body.appendChild(element);
+
+            getPicklistValuesAdapter.emit(mockGetPicklistValues);
+
+            return Promise.resolve(() => {
+                const checkboxEls = element.shadowRoot.querySelectorAll(
+                    'lightning-input'
+                );
+                expect(checkboxEls.length).toBe(7);
```

Comment:
Magic number 7. Let's extract this from the mock data set.

Reply:
Will change in cleanup round.